docs/hydra/guides/oauth2-webhooks.mdx

```diff
@@ -0,0 +1,295 @@
+---
+id: claims-at-refresh
+title: OAuth2 webhooks
+sidebar_label: OAuth2 webhooks
+keywords:
+  - webhooks
+  - custom
+  - customize
+  - claims
+  - jwt
+  - access
+  - token
+---
+
+# Customizing claims with OAuth2 webhooks
+
+```mdx-code-block
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem'
+```
+
+You can modify aspects of the OpenID Connect and access tokens returned from Hydra's OAuth2 token endpoint. A typical use case is
+adding custom claims to the tokens issued by Hydra.
```

Comment:
```suggestion
adding custom claims to the tokens issued by Ory OAuth2/Ory Hydra.
```

lets be explicit once in the beginning and then use Ory in the rest? 
IMO this document needs to work for both Ory Network users as well as self-hosted. 
I do have some ideas how to improve this in the near future.